### ISSUE TITLE
Please document the Maven coordinates

### ISSUE THREAD

**Issue by toolforger** [antlr/stringtemplate4#62]
I searched for "stringtemplate" which gave me org.antlr:stringtemplate, which ends at the deprecated 4.0.2 version (which promptly triggered the "does not recognize the delimiters clause in files' bug and sent me on an extended bug hunt).
The mention should go on http://www.antlr.org/wiki/display/ST4/Using+StringTemplate+with+Java#UsingStringTemplatewithJava-install .

Thanks!

**Comment by atamanroman** [comment]
This information should be on the main page.

**Comment by sharwell** [comment]
@atamanroman It is [in the README.md](https://github.com/antlr/stringtemplate4#maven) file. Are you looking for it somewhere else as well?

**Comment by atamanroman** [comment]
@sharwell Yes! Conventionally most maven projects have their artifact information at the project or download page (http://www.stringtemplate.org/download.html), especially if it is not obvious (ST4). I don't wan't to have to look at the repository (be it README.md or pom.xml) just to find the correct dependency.

**Comment by parrt** [comment]
I'll add to http://www.stringtemplate.org/download.html

**Comment by toolforger** [comment]
I think the coordinates should go anywhere "installation" is mentioned - most Java people "install" via a Maven repository.
By that definition, I found https://theantlrguy.atlassian.net/wiki/display/ST4/Using+StringTemplate+with+Java#UsingStringTemplatewithJava-install to be missing Maven coordinates, too.
